rod_cutting_problem/rod_cutting.py

```diff
@@ -0,0 +1,37 @@
+"""
+    This module calculates optimized solution for rod cutting
+    by function rod_cutting() with arguments as defined
+    in main()
+"""
+
+
+def rod_cutting(price, rod_len):
```

Comment:
I thought so, but It's not always necessary that one wants to know about solution for whole rod, they may want to know about piece of rod, so I kept that option open here ?
Should I remove `rod_len` ?